pallets/parachain-staking/src/benchmarks.rs

```diff
@@ -1222,40 +1285,6 @@ mod benchmarks {
 		Ok(())
 	}
 
-	#[benchmark]
-	fn execute_revoke_delegation() -> Result<(), BenchmarkError> {
```

Comment:
Why are we removing the benchmark on for this extrinsic?

Reply:
@RomarQ  Seems to be unused